src/index.spec.ts

```diff
@@ -6,15 +6,21 @@ import {
   IConfigurationStore,
   Flag,
   VariationType,
+  IBanditEvent,
+  IBanditLogger,
 } from '@eppo/js-client-sdk-common';
+import { ContextAttributes } from '@eppo/js-client-sdk-common/dist/types';
 import * as td from 'testdouble';
 
-import apiServer, { TEST_SERVER_PORT } from '../test/mockApiServer';
+import apiServer, { TEST_BANDIT_API_KEY, TEST_SERVER_PORT } from '../test/mockApiServer';
```

Comment:
This test actually spins up a server vs mocking `fetch` responses. Since different tests need different UFC files served, I'm switching off the provided SDK key.

Any client initialized with this key will be served the UFC file with bandit flags.